pkg/mocks/mock_discovery.go

```diff
@@ -0,0 +1,215 @@
+/*
```

Comment:
same here. Can you please move this to `internal/test`?

Reply:
okay 👍

Reply:
I've moved the file now 👍